.github/workflows/IntegrationTest.yml

```diff
@@ -16,12 +16,12 @@ jobs:
     name: "IntegrationTest"
     strategy:
        matrix:
-         repo:
-           - 'ITensor/ITensorBase.jl'
-           - 'ITensor/NamedDimsArrays.jl'
-           - 'ITensor/QuantumOperatorDefinitions.jl'
-           - 'ITensor/TensorAlgebra.jl'
+         pkg:
+           - 'ITensorBase'
+           - 'NamedDimsArrays'
+           - 'QuantumOperatorDefinitions'
+           - 'TensorAlgebra'
```

Comment:
```suggestion
```
As of https://github.com/ITensor/TensorAlgebra.jl/pull/27 TensorAlgebra.jl no longer using BlockSparseArrays.jl in the tests so it doesn't make sense to run it as a downstream test.